### ISSUE TITLE
In a typecript environment, @babel/parser cannot read doc if type is inserted in consts.

### ISSUE THREAD

**Issue by daybrush** [babel/babel#9246]
## Bug Report

**Current Behavior**
The following code has been passed to the ```@babel/parser```.
* no type
```ts
/**
* @namespace Consts
*/


/**
* @memberof Consts
*/
export a = true;
```
leadingComments has ```@namespace``` and ```@memberof``` doc

* but with type
```ts
/**
* @namespace Consts
*/


/**
* @memberof Consts
*/
export a: boolean = true;
```
leadingComments has only ```@memberof``` doc

**Expected behavior/code**
```ts
/**
* @namespace Consts
*/


/**
* @memberof Consts
*/
export a: boolean = true;
```
leadingComments shall have both ```@namespace``` and ```@memberof``` doc

**Environment**
- Babel version(s): 7.2.3
- Node/npm version: [e.g. Node 8/npm 5]
- OS: [e.g. OSX 10.13.4, Windows 10]
- Monorepo: [e.g. yes/no/Lerna]
- How you are using Babel: [e.g. `cli`, `register`, `loader`]

**Possible Solution**

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by kevinbarabash** [comment]
I ran into this as well.  It happens for all `VariableDeclarations`.  I ended up grabbing the missing comments from the root `File` node and attaching them manually.

**Comment by JennieJi** [comment]
Hi @kevinbarabash , this issue is fixed since v7.4.5, suggest to close
